usaspending_api/awards/v2/views/award_spending.py

```diff
@@ -52,6 +56,48 @@ class RecipientAwardSpendingViewSet(DetailViewSet):
 
     serializer_class = RecipientAwardSpendingSerializer
 
+    # Overwriting list function
+    # Some awards are insurance based and need to be grouped into the 'other' category
+    # Unfortunately we can only modify the results after the queryset has run and before
+    #  it gets put into a response. It is entirely copied from the superclass except for
+    #  the award_category lines
+    # TODO: Find a better solution to lump insurance into the other category without
+    #       modifying the database.
+    @cache_response()
+    def list(self, request, *args, **kwargs):
+        try:
+            # First, get the req object from RequestCatalog
+            created, self.req = RequestCatalog.get_or_create_from_request(request)
```

Comment:
I would try to utilize this overwritten function without the use of the RequestCatalog. We don't want to rely on a legacy feature that doesn't work consistently.